### DISCUSSION TITLE
Cross seems to ignore the features I enabled for a crate

### DISCUSSION THREAD

**Discussion by beeb** [cross-rs/cross#1092] [Q&A] [answered]
I have a project using the [ethers](https://crates.io/crates/ethers) crate that I'm trying to compile for a `x86_64-pc-windows-gnu` target from my Linux WSL2 dev environment.

My Cargo.toml file contains the following:

```toml
[dependencies]
ethers = { version = "0.17", features = ["ipc", "rustls"] }
```

I can compile for linux without any issues using `cargo build --release`. However, when I compile with `cross build --target x86_64-pc-windows-gnu`, I see the following error:

```
error[E0412]: cannot find type `Ipc` in this scope
  --> src/lib.rs:23:71
   |
23 | type IpcMiddleware = NonceManagerMiddleware<SignerMiddleware<Provider<Ipc>, Wallet<SigningKey>>>;
   |                   -                                                   ^^^ not found in this scope
   |                   |
   |                   help: you might be missing a type parameter: `<Ipc>`

error[E0599]: no function or associated item named `connect_ipc` found for struct `ethers::providers::Provider` in the current scope
  --> src/lib.rs:46:37
   |
46 |     let provider_result = Provider::connect_ipc(socket.as_path()).await;
   |                                     ^^^^^^^^^^^ function or associated item not found in `ethers::providers::Provider<_>`
```

This seems to indicate that the `ipc` feature was not activated at compile time of the ethers crate somehow.

**Comment by Emilgardis** [comment, marked as answer]
the crate seems to only support `Ipc` on unix

https://docs.rs/ethers-providers/0.17.0/src/ethers_providers/transports/mod.rs.html#22

this shouldn't be a `cross` issue

**Reply by beeb** [reply]
That totally makes sense! Thanks
